### PR TITLE
support queueing whole youtube playlists at a time

### LINKED ISSUE #6: Distinguish between "not a youtube ID" and "there was an error with this youtube ID"
When you enter the youtube ID of a video that's unavailable in your country, youtube-dl fails with an error message, but musicazoo doesn't examine the error and just tries again treating the ID as a search string. Instead it should notify the user of the error and not load any video.

### LINKED ISSUE #7: Search for and load entire playlists from youtube

### PR DESCRIPTION
**Pull request by jmgrosen**
Closes #7

### CODE REVIEW

File: webserver.py [hunk 1]
```diff
@@ -254,14 +262,16 @@ def index(self):
...
-		if not youtube_id:
+		youtube_ids = query_search(youtube_id) if youtube_id else None
+		if not youtube_ids:
 			return json.dumps({"success": False})
-		redis.rpush("musicaqueue", json.dumps({"ytid": youtube_id, "uuid": str(uuid.uuid4())}))
-		redis.rpush("musicaload", youtube_id)
-		redis.incr("musicacommon.%s" % youtube_id)
-		redis.sadd("musicacommonset", youtube_id)
-		redis.set("musicatime.%s" % youtube_id, time.time())
+		print(youtube_ids)
```
Comment (celskeggs) [inline webserver.py]: Is this print statement necessary?

File: webserver.py [hunk 2]
```diff
@@ -182,20 +182,28 @@
 </html>
 """
 
+YOUTUBE_DL = os.path.join(os.getenv("HOME"), ".local", "bin", "youtube-dl")
+
+devnull = open("/dev/null", "wb")
```
Comment (celskeggs) [inline webserver.py]: Perhaps: https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL

**Comment by jmgrosen** [comment]
* I tagged the wrong issue, oops.
* I haven't tested it extensively. I've also only tested it with the webserver component, but I think that's all that needs testing here. (It seems to do the right stuff to Redis.)

**Comment by celskeggs** [comment]
Close enough.

**Pull request merged**